nox/virtualenv.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, bin=None, env=None):
             self.env.pop(key, None)
 
         if self.bin:
-            self.env['PATH'] = ':'.join([self.bin, self.env.get('PATH')])
+            self.env['PATH'] = os.pathsep.join(
+                [self.bin, self.env.get('PATH')])
```

Comment:
@theacodes Do you want this to blow up if `PATH` isn't in the environment variables? `join()` **will** blow up (`TypeError`) if `None` is returned. Is a `KeyError` from `self.env['PATH']` preferable?

Reply:
Hrm PATH being unset is probably a pretty bizarre scenario, but according to [this](http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap08.html#tag_08_03):

> If PATH is unset or is set to null, the path search is implementation-defined.

So maybe we can just treat it like an empty string if it's not present?

Reply:
Done.